src/drivers/mpu6000/mpu6000.h

```diff
@@ -204,12 +204,12 @@
 #define MPU6000_ACCEL_DEFAULT_DRIVER_FILTER_FREQ	30
 
 #define MPU6000_GYRO_DEFAULT_RANGE_G				8
-#define MPU6000_GYRO_DEFAULT_RATE					1000
+#define MPU6000_GYRO_DEFAULT_RATE					8000
```

Comment:
No, if it is really outputting samples at 8K,  it is worse than that. The offload rate of the driver is still 1Khz. So the FIFO will be filling ~8 repeated accel samples and 8 gyro samples. The majority of the accel samples will be duplicates, but we have to read the FIFO or it will be over run. The time to off load the FIFO for the other part was ~600+Us out of 1000Us. In that time no other interrupts will be processed. 

So all the interrupt driven IO (serial transmit, USB) performance will be impacted.  Even the HRT will not provide reasonable scheduling and possibly improper time keeping.  So top output may be may be unreliable. 

Do you have a scope? Use the PROBE_INIT() and PROBE() macros to see the rate and duration of the MPU6000::measure_trampoline.